app/src/main/java/waiter/Router.java

```diff
@@ -1,23 +1,31 @@
 package waiter;
 
+import java.util.AbstractMap;
 import java.util.Map;
 
 public class Router {
 
     private final Map<String, Map<String, Map<String, String>>> routes;
     private final String noMethod = "No Method";
     private final String noUrl = "No Url";
+    Map.Entry<String, Map<String, String>> simpleGet = new AbstractMap.SimpleEntry<>("/simple_get", setResponseFields("200 OK", "Content-Length:", ""));
```

Comment:
Nitpick to make it a bit easier to read:
```java
Map.Entry<String, Map<String, String>> simpleGet = new AbstractMap.SimpleEntry<>(
  "/simple_get",
  setResponseFields(...)
);

Map.Entry<String, Map<String, String>> noUrlFound = new AbstractMap.SimpleEntry<>(
  noUrl,
  setResponseFields(...)
);
```

Reply:
Also for future consideration, create a wrapper around AbstractMap so that we don't need to expose the underlying type of `Map.Entry<String, Map<String, String>>` and it can just be:
```Java
Route simpleGet = new Route("/simple_get", setReponseFields(...));
```

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/6fd747773a9adb7e5d4ef5e1a21cfb63ff761dcb)